File: packages/colorized-brackets/test/fixtures/java/generic.java
public class Box<T> {
  //                Y @colors
  // note: the angle brackets are not highlighted
  private T t;

  public void set(T t) { this.t = t; }
  //             P   P P             P @colors
  public T get() { return t; }
  //          PP P           P @colors
}
// @colors 0=Y

File: packages/colorized-brackets/test/fixtures/rust/generic.rs
fn largest<T>(list: &[T]) -> &T {
  //         Y       P PY       Y @colors
  // note: the angle brackets are not highlighted
  let mut largest = &list[0];
  //                     P P @colors

  for item in list {
    //             P @colors
    if item > largest {
      //              B @colors
      largest = item;
    }
    // @colors 4=B
  }
  // @colors 2=P

  largest
}
// @colors 0=Y
